Cogs.Publishers/JsonSchemaProp.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Cogs.Publishers
+{
+    public class JsonSchemaProp
+    {
+        public string name; //TODO: implement own serializer
+        public string type;
```

Comment:
Lets convert these to get and set properties, and use camel casing. Such as

public string Type { get; set; }

We can do this in both of these classes